### ISSUE TITLE
Custom Recipes Not Loading

### ISSUE THREAD

**Issue by mercurialmusic** [Zrips/CMI#1084]
**Description of issue or feature request:**
Have a weird thing happening with custom recipes -- has been happening for quite a while, really -- where they're saving correctly, but they won't load with server start or even a /cmi reload. Players can't craft the custom recipe after a server restart until I enter the GUI to edit each recipe individually and click save, then they start working again. They do work properly after that point, until the server restarts again.

---

**ERROR (DELETE IF YOU HAVE NO ERROR):**
```
No console errors.
```
---

**Cmi Version (using`/cmi version`):** 
CMI plugin version: 8.0.23.0
Vault version: 1.6.11(+)
**Server Type (Spigot/Paperspigot/etc):** 
Spigot
**Server Version (using `/ver`):** 
git-Spigot-5696c83-c2035aa (MC: 1.13.2) (Implementing API version 1.13.2-R0.1-SNAPSHOT)
**Relevant plugins (Delete if this isn't needed):** 
I have no other crafting plugins installed.

**Comment by Zrips** [comment]
Is this still a problem and do you have example of recipe which doesn't load correctly for you?

**Comment by mercurialmusic** [comment]
Yes, still happening with CMI 8.2.2.0. It's affecting all of our custom recipes, our recipes.yml here: https://pastebin.com/WHdynCaK

It's quite weird. 
After restart: https://i.imgur.com/OmwvsNX.png
Editing the recipe: https://i.imgur.com/gbnoUko.png
After hitting "Update" (without making any changes): https://i.imgur.com/Obbia6M.png

No problems with any of them after hitting "Update" on each. The file itself seems fine, not losing any data, no problems loading it, etc.

**Comment by Zrips** [comment]
Strange. All works just fine for me. Will relocate recipe loading to plugins end. Maybe that will help out to push out recipes into minecraft database. Other then this, currently i have no clue what could caused this strange issue. Test out next update and if issue persists, reopen ticket.

**Comment by mercurialmusic** [comment]
Still happening unfortunately, although I'm about 90% sure it's another plugin interfering or some setting I've messed. 8.2.4.0 **fixed the issue on two of our servers** but it's still happening on the main server (the one I posted the config from), and I haven't been able to find it yet while going through plugins on a test server. It's very strange. Going to try regenerating that file, too, to see if it does anything.
